infrastructure/variables.tf

```diff
@@ -1,6 +1,10 @@
-variable "product" {}
+variable "product" {
+  default = "et"
+}
 
-variable "component" {}
+variable "component" {
+  default = "sya"
```

Comment:
Is this definitely the component name? Rather than sya-api

Reply:
sya is the frontend component name I think

Reply:
Resolved.